js/views/organization.vue

```diff
@@ -258,6 +274,8 @@ export default {
                     class: 'danger',
                     label: this._('Deleted')
                 }];
+            } else {
+                this.badges = [];
```

Comment:
I don't think this is related to this pull-request. Remove it or submit another one for this case

Reply:
![image](https://user-images.githubusercontent.com/9539326/46245474-fe873100-c3e5-11e8-9538-65529161da89.png)
The purpose of this line is to clear the "Deleted" red badge.

Reply:
OK I understand :+1: 
But on some datasets/reuses/organization, there is more than one badge (more than only the deleted badge), you may need to do some filtering to only remove the deleted badge